src/zdns/resolver.go

```diff
@@ -88,15 +89,16 @@ type ResolverConfig struct {
 	FollowCNAMEs          bool         // whether iterative lookups should follow CNAMEs/DNAMEs
 	DNSConfigFilePath     string       // path to the DNS config file, ex: /etc/resolv.conf
 
-	DNSSecEnabled       bool
-	DNSOverHTTPS        bool           // whether to use DNS over HTTPS for External Lookups, n/a to Iterative Lookups
-	DNSOverTLS          bool           // whether to use DNS over TLS for External Lookups, n/a to Iterative Lookups
-	RootCAs             *x509.CertPool // Root CAs for DoT/DoH Server Verification
-	VerifyServerCert    bool           // Verify server certificates for DoT/DoH
-	HTTPSClientIPv4     *http.Client   // for DoH, per docs should be shared amongst requests
-	HTTPSClientIPv6     *http.Client   // for DoH, per docs should be shared amongst requests
-	EdnsOptions         []dns.EDNS0
-	CheckingDisabledBit bool
+	DNSSecEnabled        bool
+	ShouldValidateDNSSEC bool           // whether to validate DNSSEC
+	DNSOverHTTPS         bool           // whether to use DNS over HTTPS for External Lookups, n/a to Iterative Lookups
+	DNSOverTLS           bool           // whether to use DNS over TLS for External Lookups, n/a to Iterative Lookups
+	RootCAs              *x509.CertPool // Root CAs for DoT/DoH Server Verification
```

Comment:
Do we want to allow people to pass in root store on the command line? Otherwise, where are we expecting it to originate?